draft-tigress-general-requirements.md

```diff
@@ -182,9 +182,11 @@ sequenceDiagram
 - (Req-Privacy) Transport protocol used to transfer provisioning information ( e.g. secure E2E transfer protocol or intermediary server) shall prevent from correlating users between exchanges or create a social graph of users involved into transfer. Intermediary server shall not be an arbiter of identity. User identities shall not be collected, stored and used for purpose other then the credential transfer itself.
 - (Req-Connectivity) Sender and Receiver shall be allowed to be online at different times. Sender and Receiver shall not need to be online at the same time. This requirement allows devices to connect to network to only exchange the portion of information required during the transfer, allowing them upload or download data in turns to network servers.
 - (Req-RoundTrips) Solution shall allow for multiple data exchanges between sender and receiver devices in the process of credential transfer. This requirement shall alighn with (Req-Connectivity) above.
+- (Req-Speed) When both Sender and Receiver are online at the same time they should be able to quickly and efficiently transfer data.
 - (Req-Opaque) In the case when an intermediary server is used to facilitate the credential transfer, message content between sender and receiver must be opaque to an intermediary, intermediary server shall not be able to recognize the content of provisioning information or use it to provision digital credential on its own.
 - (Req-SenderTrust) In the case when an intermediary server is used to facilitate the credential transfer, sender device should establish trusted relationship with the intermediary server. Intermediary server shall be able to verify that the sender device is in good standing and content generated by the sender device can be trusted by the intermediary. The trust mechanism could be proprietary or publicly verifiable ( e.g. WebAuthN). This is important because intermediary server shall have no visibility to the content of the provisiong information sent through it (Req-Opaque).
 - (Req-ReceiverTrust) In the case when an intermediary server is used to facilitate the credential transfer, receiver device should be able to evaluate the trustworthiness of the intermediary based on agreed criteria.
+- (Req-Invitation) The Receiver must be able to establish a connection with the Sender for the secure credential transfer using an invite that can be sent over any communication channel.
```

Comment:
I am not sold on this wording, but I couldn't come up with anything better.

Reply:
We used to have Req-Preview - which was about receiver being able to see what pass they are about to accept

Reply:
You could incorporate this with the X-Platform requirement saying the transfer and invite should be able to occur on any platform and any communication channel.

I think the intention here is slightly different than the Req-Preview if I understand correctly.